### PR TITLE
README file design update phase 1 and other minor changes

### PR DESCRIPTION
**Pull request by mabdullahabid**
There hasn't been an update to the `master` branch of django-scaffold so I figured we should update it as we've already updated the `crowdbotics-slack-app` and `modules` repo.

- minor changes from the previous month
- few dependabot updates
- README update

### CODE REVIEW

File: {{cookiecutter.project_slug}}/modules/urls.py [hunk 1]
```diff
@@ -11,12 +12,13 @@
 
 try:
     modules_dir = f"{settings.BASE_DIR}/modules/"
-    urls = Path(modules_dir).rglob('urls.py')
+    urls = Path(modules_dir).rglob("urls.py")
     for url in urls:
-        module_name, _ = url.as_posix().split('/')[-2:]
+        module_name, _ = url.as_posix().split("/")[-2:]
         if not module_name == "modules":
+            module_url = module_name.replace("_", "-")
```
Comment (driverdan) [inline {{cookiecutter.project_slug}}/modules/urls.py:19]: The file system should have underscore too, not dashes. Why is this needed?

**Review by driverdan — CHANGES_REQUESTED** [review]

File: {{cookiecutter.project_slug}}/Pipfile [hunk 2]
```diff
@@ -28,5 +28,5 @@ drf-yasg = "~=1.20.0"
 fcm-django = "~=0.3.10"
 pyyaml = "~=5.4.1"
 pillow = "~=8.2.0"
-pytest = "==6.2.3"
+pytest = "==6.2.4"
```
[the remark this answers is not in the document]
Comment (mabdullahabid) [reply {{cookiecutter.project_slug}}/Pipfile:31]: I believe these are from dependabot here https://github.com/crowdbotics/django-scaffold/pull/219/commits/84ac43adc211b8b12046050150ef7c9835495fec. We haven't had a release since May 6th so all the changes have been accumulating in develop.

File: {{cookiecutter.project_slug}}/modules/urls.py (hunk 1 above)
Comment (mabdullahabid) [reply {{cookiecutter.project_slug}}/modules/urls.py:19]: @danielsousaio would be more suited to answer this https://github.com/crowdbotics/django-scaffold/pull/219/commits/5b1c16a8addf3bff0fb66c1c4d8bbd4ab4b16790

File: {{cookiecutter.project_slug}}/Pipfile (hunk 2 above)
[the remark this answers is not in the document]
Comment (mabdullahabid) [reply {{cookiecutter.project_slug}}/Pipfile:31]: `develop` is set as the main branch for this repo so I guess dependabot sends all the PRs to `develop` instead of `master`

**Comment by driverdan** [comment]
Sorry, I didn't realize this was a merge to master. I thought this was a regular PR.

**Review by driverdan — APPROVED** [review]
